runhouse/resources/hardware/cluster.py

```diff
@@ -739,18 +739,45 @@ def connect_server_client(self, force_reconnect=False):
                 system=self,
             )
 
-    def status(self, resource_address: str = None):
-        """Loads the status of the Runhouse daemon running on the cluster."""
+    def status(self, resource_address: str = None, send_to_den: bool = False):
+        """
+        Loads the status of the Runhouse daemon running on the cluster.
+        :param resource_address: resource's rns address (/<owner_username>/<resource_name>)
+        :param send_to_den: If true, the loaded status will be updated in den as well. Default: False.
+        :return: a dictionary, containing info about cluster status.
+        """
+        # TODO [sb]: add to docs the response schema of the status output, so users will know what to expect.
         # Note: If running outside a local cluster need to include a resource address to construct the cluster subtoken
         # Allow for specifying a resource address explicitly in case the resource has no rns address yet
         if self.on_this_cluster():
-            status = obj_store.status()
+            status = obj_store.status(send_to_den=send_to_den)
         else:
             status = self.call_client_method(
                 "status",
                 restart_server=False,
                 resource_address=resource_address or self.rns_address,
+                send_to_den=send_to_den,
             )
+
+        # don't want to return it as part of the status, it is for internal usage only
+        status = dict(status) if not isinstance(status, dict) else status
+        send_to_den_resp = status.pop("send_to_den_resp")
```

Comment:
why do we need to pop this / include it in the reponse data to begin with? don't we already know its value since it's passed into this function

Reply:
`send_to_den_resp` contains info that we want to log, in case the send to den was not successful. See more about that in my comment below. :)